### PR TITLE
Rewrite the trusted_certs directory documentation

### PR DESCRIPTION
**Pull request by tas50**
It wasn't correct for client and didn't mention our resource.

Signed-off-by: Tim Smith <tsmith@chef.io>

### CODE REVIEW

File: content/chef_client_security.md [hunk 1]
```diff
@@ -58,17 +58,23 @@ to on-premises Chef Infra Server.
...
-
--   On each workstation, this directory is the location into which SSL
-    certificates are placed after they are downloaded from the Chef
-    Infra Server using the `knife ssl fetch` subcommand
--   On every node, this directory is the location into which SSL
-    certificates are placed when a node has been bootstrapped with Chef
-    Infra Client from a workstation
+### trusted_certs Directory
+
+Chef Infra Client and Chef Workstation store additional trusted SSL
```
Comment (kagarmoe) [inline content/chef_client_security.md]: What do you mean by 'additional' ?

Comment (tas50) [reply content/chef_client_security.md]: It's a cert or a CA that's not in the cacert.pem bundle. That might be your companies own CA or some self signed cert.

File: content/chef_client_security.md [hunk 2]
```diff
@@ -60,21 +60,21 @@ to on-premises Chef Infra Server.
...
-In Chef Workstation the `trusted_certs` directory is located at `~/.chef/trusted_certs`
-by default. Running `knife ssl fetch` fetches the current Chef Infra Server certificate, 
-and saves the certificate to the `~/.chef/trusted_certs` directory automatically.
-
-In Chef Infra Client the trusted certificates directory is found at `C:\chef\trusted_certs`
-on Windows and `/etc/chef/trusted_certs` on all other platforms. Certificates can be placed
-in these directories as part of bootstrap process and they can be managed continuously
-using the [chef_client_trusted_certificate]({{< relref "/resources/chef_client_trusted_certificate" >}})
-Chef Infra Client resource.
+Your organization may use self-signed SSL certificates in internal networks or during software development and testing.
```
Comment (tas50) [inline content/chef_client_security.md]: you could have your own CA as well

File: content/chef_client_security.md [hunk 3]
```diff
@@ -60,21 +60,21 @@ to on-premises Chef Infra Server.
...
+
+The `trusted_certs` directory on Chef Infra Client and Chef Workstation that works as a trusted certificate store for any SSL certificates needed to access Chef Infra Server. Chef Infra Server trusts all certificates stored in this directory--including certificates that are not issued by a trusted Certificate Authority. Place self-signed certificates in the `trusted_certs` directory to use them on Chef Infra Servers and use this directory for other certificates needed by the applications and services in your Chef-managed system.
+
+The `trusted_certs` directory is located on Chef Workstation at:
+
+* Windows: ``C:\chef\trusted_certs`
+* Linux/macOS: `/.chef/trusted_certs`
```
Comment (tas50) [inline content/chef_client_security.md]: Unix-y stuff too like Solaris, FreeBSD, and AIX. This is really non-Windows stuff

File: content/chef_client_security.md (hunk 2 above)
Comment (kagarmoe) [reply content/chef_client_security.md]: I wasn't going to talk about private CA, but I could

Comment (tas50) [reply content/chef_client_security.md]: That's the legit use of the feature so we should call it out

Comment (kagarmoe) [reply content/chef_client_security.md]: See the update

File: content/chef_client_security.md (hunk 3 above)
Comment (kagarmoe) [reply content/chef_client_security.md]: OK

File: content/chef_client_security.md [hunk 4]
```diff
@@ -52,22 +52,36 @@ The following information does not apply to Hosted Chef, only to on-premises Che
...
-### `/.chef/trusted_certs`
+### trusted_certs Directory
 
-The `/.chef/trusted_certs` directory stores trusted SSL certificates
-used to access the Chef Infra Server:
+Your organization may use a private Certificate Authority (CA) to generate SSL Certificates or they may create self-signed SSL certificates to use on internal networks or during software development and testing.
 
-* On each workstation, this directory is the location into which SScertificates are placed after they are downloaded from the CheInfra Server using the `knife ssl fetch` subcommand
-* On every node, this directory is the location into which SSLcertificates are placed when a node has been bootstrapped with ChefInfra Client from a workstation
+The `trusted_certs` directory on Chef Workstation and in Chef Infra Client works as a trusted certificate store for any SSL certificates needed to access Chef Infra Server. Chef Infra Server trusts all certificates stored in this directory--including certificates that are not issued by a trusted Certificate Authority (CA).
```
Comment (tas50) [inline content/chef_client_security.md]: It's used for any communication the client or workstation makes. So if you have a Chef cookbook accessing remote websites in say a `remote_file` resource trusted_certs will handle the certs on that site.

File: content/chef_client_security.md [hunk 5]
```diff
@@ -52,22 +52,36 @@ The following information does not apply to Hosted Chef, only to on-premises Che
...
+
+#### trusted_certs Locations
+
+When you install Chef Workstation, it creates a `trusted_certs` directory.
+The Chef Workstation `trusted_certs` directory is located:
+
+- Windows: `C:\chef\trusted_certs`
+- All other systems: `/.chef/trusted_certs`
+
+When you bootstrap a node, the Chef Infra Client copies the SSL certificates for the Chef Infra Server to `/.etc/chef`. A node `trusted_certs` directory is located at:
```
Comment (tas50) [inline content/chef_client_security.md]:
```suggestion
When you bootstrap a node, the Chef Infra Client copies the SSL certificates for the Chef Infra Server to `/etc/chef`. A node `trusted_certs` directory is located at:
```

File: content/chef_client_security.md [hunk 6]
```diff
@@ -52,22 +52,36 @@ The following information does not apply to Hosted Chef, only to on-premises Che
...
+
+- `/.etc/chef` (Chef Infra supports Linux nodes)
```
Comment (tas50) [inline content/chef_client_security.md]:
```suggestion
- `/etc/chef` (Chef Infra supports Linux nodes)
```

File: content/chef_client_security.md [hunk 7]
```diff
@@ -127,8 +141,8 @@ Use following client.rb settings to manage SSL certificate preferences:
 
 The Chef Infra Client includes two knife commands for managing SSL certificates:
 
-* Use [knife ssl check](/workstation/knife_ssl_check/) to troubleshoot SS certificate issues
-* Use [knife ssl fetch](/workstation/knife_ssl_fetch/) to pull down a certificate from the Chef Infra Server to the `/.chef/trusted_certs` directory on the workstation.
+- Use [knife ssl check](/workstation/knife_ssl_check/) to troubleshoot SS certificate issues
```
Comment (tas50) [inline content/chef_client_security.md]:
```suggestion
- Use [knife ssl check](/workstation/knife_ssl_check/) to troubleshoot SSL certificate issues
```

File: content/chef_client_security.md [hunk 8]
```diff
@@ -52,22 +52,36 @@ The following information does not apply to Hosted Chef, only to on-premises Che
...
+Place private and self-signed certificates in the `trusted_certs` directory to use them on Chef Infra Servers. Also use this directory to store other certificates needed by the applications and services in your Chef-managed system.
+
+Use the the [chef_client_trusted_certificate]({{< relref "/resources/chef_client_trusted_certificate" >}}) Chef Infra Client resource to manage these certificates continuously.
+
+#### trusted_certs Locations
+
+When you install Chef Workstation, it creates a `trusted_certs` directory.
+The Chef Workstation `trusted_certs` directory is located:
+
+- Windows: `C:\chef\trusted_certs`
```
Comment (tas50) [inline content/chef_client_security.md]: Both of these platforms are the .chef directory.

File: content/chef_client_security.md (hunk 6 above)
Comment (tas50) [reply content/chef_client_security.md]: This is where we need to call out the windows vs. non-windows platforms

Comment (kagarmoe) [reply content/chef_client_security.md]: Do we now support Windows nodes?

**Pull request merged**